includes/abstracts/class-wc-banklink-ipizza.php

```diff
@@ -231,6 +231,20 @@ function generate_submit_form( $order_id ) {
 		// Encode signature
 		$macFields['VK_MAC'] = base64_encode( $signature );
 
+		// language support: informs bank of preferred UI language
+		if ( defined( 'ICL_LANGUAGE_CODE' ) ) {
+			// WPML
+			$macFields['VK_LANG'] = 'ENG';
+			if ( ICL_LANGUAGE_CODE == 'et' ) $macFields['VK_LANG'] = 'EST';
+			if ( ICL_LANGUAGE_CODE == 'ru' ) $macFields['VK_LANG'] = 'RUS';
+		} elseif ( function_exists( 'qtrans_getLanguage' ) ) {
+			// qtranslate
+			$lang = qtrans_getLanguage();
+			$macFields['VK_LANG'] = 'ENG';
+			if ( $lang == 'et' ) $macFields['VK_LANG'] = 'EST';
+			if ( $lang == 'ru' ) $macFields['VK_LANG'] = 'RUS';
+		}
+
```

Comment:
@tjuris How about setting the VK_LANG by default to EST (and apply filter to change it when needed)? Then apply other language if multilingual plugin is used.

```php
$language = false;

if( defined ... ) {
    $language = ICL_LANGUAGE_CODE;
}
// new line
elseif( qtrans ) {
    $language = ...
}
else {
   $language = substr( get_locale(), 0, 2 );
}

if( $language !== FALSE ) {
    $language = str_ireplace( array( 'et', 'ru', 'en' ), array( 'EST', 'RUS', 'ENG' ) ); 
}
else {
    $language = 'EST'; // apply filters or make it an option
}

$macFields['VK_LANG'] = $language;
```